### PR TITLE
Add 4.12 rollback test for stable builds

### PR DESCRIPTION
**Pull request by shellyyang1989**
Refers [OTA-1084](https://issues.redhat.com/browse/OTA-1084)

### CODE REVIEW

**Comment by shellyyang1989** [comment]
/pj-rehearse ci/rehearse/periodic-ci-openshift-release-master-stable-4.12-aws-ipi-byo-route53-upgrade-rollback-oldest-supported

**Comment by shellyyang1989** [comment]
/pj-rehearse periodic-ci-openshift-release-master-stable-4.12-aws-ipi-byo-route53-upgrade-rollback-oldest-supported

**Comment by shellyyang1989** [comment]
[Rehearsal](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/46830/rehearse-46830-periodic-ci-openshift-release-master-stable-4.12-aws-ipi-byo-route53-upgrade-rollback-oldest-supported/1735993262067421184) failed due to [OCPBUGS-25382](https://issues.redhat.com/browse/OCPBUGS-25382)

File: ci-operator/config/openshift/release/openshift-release-master__stable-4.12.yaml [hunk 1]
```diff
@@ -0,0 +1,261 @@
+base_images:
```
Comment (wking) [inline ci-operator/config/openshift/release/openshift-release-master__stable-4.12.yaml:1]: We don't have anything that looks like the `openshift-release-master__stable-4.12.yaml` pattern to me.  The only existing `__stable-4.*` are 4.8 and older update jobs:

```console
$ ls ci-operator/config/openshift/release/openshift-release-master__stable-4.*.yaml
...
ci-operator/config/openshift/release/openshift-release-master__stable-4.8-upgrade-from-stable-4.4.yaml
ci-operator/config/openshift/release/openshift-release-master__stable-4.8-upgrade-from-stable-4.5.yaml
```

and the other z-rollback jobs all use `__nightly`:

```console
$ git --no-pager grep oldest-supported-z-rollback: -- ci-operator/config/openshift/release
ci-operator/config/openshift/release/openshift-release-master__nightly-4.10.yaml:  oldest-supported-z-rollback:
ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml:  oldest-supported-z-rollback:
ci-operator/config/openshift/release/openshift-release-master__nightly-4.15.yaml:  oldest-supported-z-rollback:
ci-operator/config/openshift/release/openshift-release-master__nightly-4.16.yaml:  oldest-supported-z-rollback:
ci-operator/config/openshift/release/openshift-release-master__nightly-4.7.yaml:  oldest-supported-z-rollback:
ci-operator/config/openshift/release/openshift-release-master__nightly-4.8.yaml:  oldest-supported-z-rollback:
ci-operator/config/openshift/release/openshift-release-master__nightly-4.9.yaml:  oldest-supported-z-rollback:
```

Can we use `openshift-release-master__nightly-4.12.yaml` here too, to match that pattern?

Comment (shellyyang1989) [reply ci-operator/config/openshift/release/openshift-release-master__stable-4.12.yaml:1]: QE is used to separate tests on nightly builds and stable builds into `__nightly` and `__stable` files respectively. So we can easily identify whether nightly build or stable build we are using for the test from the job name, like: 
periodic-ci-openshift-release-master-**stable**-4.12-gcp-ipi-ovn-ipsec-upgrade-rollback-oldest-supported, 
periodic-ci-openshift-release-master-**nightly**-4.14-e2e-aws-ovn-upgrade-rollback-oldest-supported.
Dev might use a different rule. If you prefer to merge the stable test into `__nightly` file, It's fine with me too.

**Comment by shellyyang1989** [comment]
Thanks Trevor for your review.

@petr-muller @LalatenduMohanty Could you please help review as well when you have time? Thank you.

**Pull request closed without merging**